lms/djangoapps/verify_student/models.py

```diff
@@ -721,8 +721,8 @@ def submit(self, copy_id_photo_from=None):
                 self.status = "must_retry"
                 self.error_msg = response.text
                 self.save()
-        except Exception as error:
-            log.exception(error)
+        except Exception:
+            log.exception("Request sending to Software Secure failed, setting status to must_retry")
```

Comment:
@noraiz-anwar I have added the trace in the parent ticket. I am not sure if the user information is logged during the exception. Please verify